pkg/object/statussynccontroller/statussynccontroller.go

```diff
@@ -192,29 +231,34 @@ func (ssc *StatusSyncController) handleStatus(unixTimestamp int64) {
 }
 
 func (ssc *StatusSyncController) syncStatusToCluster(statuses map[string]string) {
-	kvs := make(map[string]*string)
-
 	// Delete statuses which disappeared in current status.
 	if ssc.lastSyncStatuses != nil {
 		for k := range ssc.lastSyncStatuses {
 			if _, exists := statuses[k]; !exists {
+				kv := make(map[string]*string)
 				k = ssc.superSpec.Super().Cluster().Layout().StatusObjectKey(k)
-				kvs[k] = nil
+				kv[k] = nil
+
+				err := ssc.superSpec.Super().Cluster().PutAndDeleteUnderLease(kv)
+				if err != nil {
+					logger.Errorf("sync status failed. If the message size is too large, "+
+						"please increase the value of cluster.MaxCallSendMsgSize in configuration: %v", err)
+				}
 			}
 		}
```

Comment:
propose to do delete in one call to PutAndDeleteUnderLease, that's refactor this piece of code to:
```go
                kv := make(map[string]*string)
		for k := range ssc.lastSyncStatuses {
			if _, exists := statuses[k]; !exists {
				k = ssc.superSpec.Super().Cluster().Layout().StatusObjectKey(k)
				kv[k] = nil
			}
		}
		err := ssc.superSpec.Super().Cluster().PutAndDeleteUnderLease(kv)
		if err != nil {
			logger.Errorf("sync status failed. If the message size is too large, "+
				"please increase the value of cluster.MaxCallSendMsgSize in configuration: %v", err)
		}
```